app/pb_kits/playbook/pb_image/_image.jsx

```diff
@@ -11,6 +11,8 @@ type ImageProps = {
   className?: string,
   data?: object,
   id?: string,
+  size: "xs" | "sm" | "md" | "lg" | "xl",
```

Comment:
The default should be none. The current use of the image kit doesn't resize the image that is sent. We need to keep this functionality as default.

Reply:
updated